### ISSUE TITLE
support RTL

### ISSUE THREAD

**Issue by ahmedoid** [machadogj/react-native-carousel-control#27]
ScrollView in RTL devices don't work 
```
<View style={[ styles.container, this.props.containerStyle ]}>
                <ScrollView
                    automaticallyAdjustContentInsets={ false }
                    bounces
                    contentContainerStyle={ [ computedStyles.scrollView ] }
                     style={{ flexDirection: I18nManager.isRTL ? 'row-reverse' : 'row' }}
                    decelerationRate={ 0.9 }
                    horizontal
                    onScrollEndDrag={ this._handleScrollEnd }
                    ref={ c => this.scrollView = c }
                    showsHorizontalScrollIndicator={ false }
                >
                    { body }
                </ScrollView>
            </View>
```

add this line work perfect in RTL

` style={{ flexDirection: I18nManager.isRTL ? 'row-reverse' : 'row' }}`

**Comment by machadogj** [comment]
@ahmedoid awesome point! thanks for the help 👍

**Comment by machadogj** [comment]
This has been published in v1.1.2!
